### PR TITLE
feature(component): mega menu

### PR DESCRIPTION
**Pull request by dyarmachok**
Mega menu FE implementation

Ticket: [#180](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/180)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 8629312b7dbbbb506cc4422247f21b7dca8545b8

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5ca0aa4e40905800074422b4

**Comment by gepifanio** [comment]
Hey @dyarmachok, 

I noticed a bug in the mobile version, I will attach here to clarify. 

### problem
the secondary navigation is showing behind the hamburger menu when is active on the mobile version.

https://media.giphy.com/media/vvV7CuQs4o73gDeltN/giphy.gif

**Comment by jaredh-um** [comment]
Hi @gepifanio @TimMansell - some feedback:

File: components/page/megamenu/MegaMenu.vue [hunk 1]
```diff
@@ -67,63 +74,81 @@
...
-                      v-if="rootitem.feature.alt"
-                      class="screenreaders-only"
-                      v-html="rootitem.feature.alt"
-                    />
-                  </component>
+                  />
+                  <div
+                    v-if="!isMobileOpen"
+                    class="menu__aside-info">
+                    <p class="menu__aside-info__text">Dance 40 at VCA celebrates lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris vitae felis id tortor congue dapibus. In pulvinar semper dolor. Sed non mi non arcu dictum hendrerit. Nulla partititor sed rcu nec malesuada </p>
```
Comment (gepifanio) [inline components/page/megamenu/MegaMenu.vue]: Static text

File: components/page/megamenu/_mega-menu-title.css [hunk 2]
```diff
@@ -1,10 +1,15 @@
 .page-title {
-  margin: 0 1.5em;
+  margin: 0 1.25em;
+
+  .active & {
```
Comment (TimMansell) [inline components/page/megamenu/_mega-menu-title.css]: This moves the page title when mega menu link is hovered

**Comment by gepifanio** [comment]
Issues:

- [ ] IE 11 cut the search part a little bit
- [ ] Mobile the hamburger menu sub-items shows on top of the name of the faculty
- [ ] If add too many subitems do not go to the next column only increase the size of the submenu.

**Pull request merged**